blocks/url-input/button.js

```diff
@@ -15,61 +16,184 @@ import { IconButton } from '@wordpress/components';
  * Internal dependencies
  */
 import UrlInput from './';
+import ToggleControl from '../inspector-controls/toggle-control';
+import { filterURLForDisplay } from '../../editor/utils/url';
+
+const DisplayStep = 'DISPLAY';
+const EditStep = 'EDIT';
+const SettingsStep = 'SETTINGS';
+
+const AllSteps = [ EditStep, SettingsStep, DisplayStep ];
```

Comment:
We have this unofficial style guideline for uppercasing module constants? Could we uppercase those?

`const DISPLAY_STEP`...